test/server-test.js

```diff
@@ -23,5 +25,45 @@ describe('Server', function(){
     assert(app)
   })
 
-  
+  describe('GET /api/foods', function() {
+    this.timeout(1000000)
+    beforeEach(function(done) {
+      Food.createFood("Banana", 105)
+      .then(function() {
+        Food.createFood("French Silk Pie", 340).then(function() { done() })
+      });
+    })
+
+    afterEach(function(done) {
+      Food.emptyFoodsTable().then(function() {
+        done()
+      })
+    })
+
+    it('should have two food items from the resource', function(done) {
+      var ourRequest = this.request
+      Food.findAllFoods().then(function(data) {
```

Comment:
Yes! In class we did it this way, but I actually prefer to not do the database call. Going to update that now.